src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -51,17 +53,17 @@ public void storeMessage(Message message) {
   }
 
   /**
-   * Gets messages posted by a specific user.
+   * Gets messages received by a specific user.
    *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
+   * @return a list of messages received by the user, or empty list if user has never received a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
+  public List<Message> getMessages(String recipient) {
```

Comment:
For clarity, you may want to consider renaming the function; for example "getMessagesSentTo" or "getMessagesReceivedBy", which would clarify that this function returns messages received by the given user rather than all messages which are visible to the given user. Alternatively, you may want to consider changing the definition/implementation so that this returns all messages that are visible to the given user, including both messages that the given user has sent as well as public messages (with no recipient) and those sent to this user.

Reply:
Okay I changed the naming of the function to make it more intuitive. I kept the logic of the function the same though because this is intended to display all the messages a person has received on their page, kind of like a discussion board. This function doesn't get used though since I'm using the function "getMessagesBetweenTwoUsers" instead to implement direct messaging, so maybe I should just delete this function entirely?

Reply:
If the function is not used, then I would simply delete it.